ci-operator/config/openshift/networking-console-plugin/openshift-networking-console-plugin-main__okd-scos.yaml

```diff
@@ -26,12 +26,12 @@ images:
   to: networking-console-plugin
 promotion:
   to:
-  - name: scos-4.17
+  - name: "4.18"
```

Comment:
scos-4.x is the appropriate integration stream for OKD. "4.x" was for the old fcos builds and is deprecated.

Reply:
just for the record, initially i tried putting `scos-4.18`, but the `openshift-image-mirror-mappings` check deleted the entry in question. however, it turns out that actually pushing the image to `quay.io/openshift-origin-*` is not required.